### DISCUSSION TITLE
Changing SYSFS entries

### DISCUSSION THREAD

**Discussion by PCBscanner** [berarma/new-lg4ff#94] [Q&A] [answered]
I've installed the module as-is and can confirm it is working with a G25 wheel and pedals, however I'd like to adjust some of the setting such as gain, autocenter, spring_level etc. 

I'm new to working with drivers, and I've tried looking around but didn't know where to start with changing these values. Is there a guide that I've missed with how to e.g., set autocenter to 20,000? Would I need to change a variable in the various .c files and re-make?

Cheers,

PCB

**Comment by berarma** [comment, marked as answer]
You can use [Oversteer](/berarma/oversteer). It's a gui tool for that purpose. Changing the sysfs entries manually is a bit cumbersome.

**Reply by PCBscanner** [reply]
Thanks Berarma. Now to spend time dialling it in!
